tests/rptest/scale_tests/large_messages_test.py

```diff
@@ -0,0 +1,488 @@
+                # 'num.replica.fetchers': 16,
+                # 'replica.fetch.max.bytes': 10485760 * 2,
+                # 'num.network.threads': 16,
+                # 'num.io.threads': 16,
+            }
+            if self.scale.local_retention_bytes:
+                config['retention.local.target.bytes'] = \
+                        self.scale.local_retention_bytes
+
+            self.rpk.create_topic(tn,
+                                  partitions=self.n_partitions,
+                                  replicas=self.replication_factor,
+                                  config=config)
+
+    def _wait_until_cluster_healthy(self, include_underreplicated=True):
+        """
+        Waits until the cluster is reporting no under-replicated
+        or leaderless partitions.
+        """
+        def is_healthy():
+            unavailable_count = self.redpanda.metric_sum(
+                'redpanda_cluster_unavailable_partitions',
+                metrics_endpoint=MetricsEndpoint.PUBLIC_METRICS,
+                nodes=self.redpanda.started_nodes())
+            under_replicated_count = self.redpanda.metric_sum(
+                'vectorized_cluster_partition_under_replicated_replicas',
+                nodes=self.redpanda.started_nodes())
+            self.logger.info(
+                f"under-replicated partitions count: {under_replicated_count} "
+                f"unavailable_count: {unavailable_count}")
+            return unavailable_count == 0 and \
+                (under_replicated_count == 0 or not include_underreplicated)
+
+        wait_until(
+            lambda: is_healthy(),
+            timeout_sec=self.HEALTHY_WAIT_SECONDS,
+            backoff_sec=30,
+            err_msg=f"couldn't reach under-replicated count target: {0}")
+
+    def _run_producers_with_constant_rate(self, topic_count, topic_prefixes,
+                                          message_rps):
+        swarm_node_producers = []
+        for topic in topic_prefixes:
+            swarm_producer = ProducerSwarm(
+                self.test_context,
+                self.redpanda,
+                topic,
+                topic_count,
+                self.message_count,
+                unique_topics=True,
+                messages_per_second_per_producer=message_rps,
+                min_record_size=self.message_size,
+                max_record_size=self.message_size)
+            swarm_node_producers.append(swarm_producer)
+
+        # Run topic swarm for each topic group
+        for swarm_client in swarm_node_producers:
+            self.logger.info(f"Starting swarm client on node {swarm_client}")
+            swarm_client.start()
+
+        return swarm_node_producers
+
+    def _run_consumers_with_constant_rate(self, topic_count, topic_prefixes,
+                                          group):
+        swarm_node_consumers = []
+        node_message_count = int(0.95 * (self.message_count * topic_count))
```

Comment:
Account for 5% errors as we are testing limits and RP might fail to do so. It can be 100% actually, I am just using the same approach and logic we used for many_topics_test suite

Reply:
I don't have problem with 95%, but just want to confirm.  If we try 100% and test fails to get 100%, would it be a problem? Would we want to know here?

Reply:
We account for 5 failed messages out of 100 due to networking and AWS slowdowns. Actually, if this fails - it will be a lot more than 5%. By this lesser amount we remove flakiness introduced by minor slowdowns. Also, critical part here is respecting throughput, not message count.